content/packages/ts-music-mirrors.txt

```diff
@@ -1,4 +1,4 @@
 http://openra.baxxster.no/openra/ts-music.zip
 http://openra.us.baxxster.no/openra/ts-music.zip
-http://81.169.246.181/ts-music.zip
+http://81.169.246.181/openra/ts/ts-tsmusic.zip
```

Comment:
This now gives "404 - File or directory not found." Shall we remove it or can you fix it server side?